pages/docs/migration-guide-4-5.md

```diff
@@ -75,6 +75,30 @@ app.get("/") { ctx ->
 {% endcapture %}
 {% include macros/docsSnippet.html java=java kotlin=kotlin %}
 
+## SSE changes
+In Javalin 4, connections to SSE clients needed to be closed manually, this resulted in SSE client leaking when clients were not properly closed.
```

Comment:
I'm not sure if I agree with (or understand) this sentence.

Reply:
What about: 
>In Javalin 4, clients weren't automatically closed, this allowed people to keep lists of clients outside of the handler.

Reply:
Ah, I see. Yes, that's better.